### ISSUE TITLE
Update post install message

### ISSUE THREAD

**Issue by eheydrick** [sensu-plugins/community#32]
Currently when you install a plugin it outputs the message `You can use the embedded Ruby by setting EMBEDDED_RUBY=true in /etc/default/sensu`. This was helpful years ago before embedded ruby was enabled by default but now it can [can be confusing](https://github.com/sensu-plugins/sensu-plugins-influxdb/issues/32#issuecomment-313704160) as the message suggests you need to take action when none is required anymore.

`EMBEDDED_RUBY=true` [was made the default](https://github.com/sensu/sensu/blob/master/CHANGELOG.md#important-3) in Sensu 0.21.0, which was released in 11/2015. The [Chef cookbook](https://github.com/sensu/sensu-chef/blob/develop/attributes/default.rb#L27) and [Puppet module](https://github.com/sensu/sensu-puppet/blob/master/manifests/init.pp#L498) default to `EMBEDDED_RUBY=true` as well. 

Instead, we should say something like "Thanks for installing Sensu Plugins!"

**Comment by majormoses** [comment]
Agreed on the whole `EMBEDDED_RUBY` message. For post install what about pointing it to IRC/slack? This gives people a quick helper to find our community communication challen.

**Comment by eheydrick** [comment]
Maybe we should have a page that has pointers to support resources and we link to that.

**Comment by majormoses** [comment]
Yup, let's discuss what we should do and close out the PR if you are good with that. Too many moving things moving right now and once the communication, website, and all that get worked out I think we should avoid making the changes as we are just doubling our work.

**Comment by majormoses** [comment]
@mbbroberg ping

**Comment by mbbroberg** [comment]
@majormoses @eheydrick is there a place in the sensu-plugin.io page this could go?

**Comment by majormoses** [comment]
atm I would guess not, the website is a mess. The builds are broken, it does not have valid ssl, and we have discussed redoing it. I feel like the order of operations on this is wrong. We should first figure that out before charging forward.

**Comment by mbbroberg** [comment]
The only site maintained is sensuapp.org. I could see us adding `/community` here and linking appropriately. Thoughts?

**Comment by majormoses** [comment]
that makes sense to me and then we can update that page later to point to the various locations of note.

**Comment by majormoses** [comment]
@eheydrick also now with sensu 2.x and the asset management stuff it probably makes even more sense. I wonder if we should start creating a shared gem for this kind of stuff so we only have to update it in one place in the future?
